lightningd/gossip/gossip.c

```diff
@@ -145,7 +145,7 @@ static void send_peer_with_fds(struct peer *peer, const u8 *msg)
 	/* Now we talk to socket to get to peer's owner daemon. */
 	peer->local = false;
 	/* FIXME: Forget peer if other end is closed. */
-	daemon_conn_init(peer, &peer->owner_conn, fds[0], owner_msg_in);
+	daemon_conn_init(peer, &peer->owner_conn, fds[0], owner_msg_in, NULL);
```

Comment:
Might be a good place to issue the disabling `channel_update` instead of doing so in the dying `channeld`, though that would also mean accessing the HSM on behalf of the channel. Will add this to my "to investigate list".